### ISSUE TITLE
Volume2 icon stops changing randomly (version 2)

### ISSUE THREAD

**Issue by jimbo1qaz** [irzyxa/Volume2#15]
Even after 12.28 and "reload hooks" enabled every 5 seconds, Volume2 stops updating its tray icon randomly.

**Comment by fifonik** [comment]
I have the similar (or the same) issue.

Sometimes the icon in system tray does not displayed on specific volume (50% that is my default volume). If I increase volume -- icon displayed correctly. If I decrease the volume back to the same level -- no icon again :(

To fix the issue I need to restart the program or go to settings and change the icon to something different, press "Apply" and then set the icon back.
It happens quite often, however, I do not know how to reproduce the issue.

**Comment by jimbo1qaz** [comment]
Are you using a high dpi display?

**Comment by fifonik** [comment]
Nope, 1920*1200. Different graphics cards (ATI/AMD) and drivers. Different audio drivers (Realtek).

I'm using keyboard shortsuts to set volume to 100% and 50% and I use them often when se watching movies (using PotPlayer).
So usually I do:
- Run player in windowed mode by double clicking video file
- Crtrl + Alt + F (my shortcut) to set system volume to 100%
- Enter to switch player to full screen
- Up/Down a few times while watching movie to increase/decrease player's audio volume 
...
- Esc to exit the player
- Ctrl + Alt + A (my shortcut) to set system volume to 50%
At this time the volume is changed to 50% and usually the Volume2 icon disappeared. If I switch to 100% again -- it's there. Back to 50% using shortcut or mouse wheel -- no icon again.
Unfortunately, this happens not every time.
